web/static.go

```diff
@@ -73,7 +76,28 @@ func root(c *Context, w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Cache-Control", "no-cache, max-age=31556926, public")
 
 	staticDir, _ := fileutils.FindDir(model.ClientDir)
-	http.ServeFile(w, r, filepath.Join(staticDir, "root.html"))
+	contents, err := os.ReadFile(filepath.Join(staticDir, "root.html"))
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusNotFound)
+		return
+	}
+
+	titleTemplate := "<title>%s</title>"
+	strToBeChanged := fmt.Sprintf(titleTemplate, model.TeamSettingsDefaultSiteName)
+	strToBeInserted := strToBeChanged
+	siteName := c.App.Srv().Config().TeamSettings.SiteName
+	ogDescriptionMetaTag := getOGDescriptionMetaTag(c)
+	if siteName != nil && model.TeamSettingsDefaultSiteName != *siteName {
```

Comment:
Thinking the same here

```suggestion
	if siteName != nil && len(*siteName) > 0 {
```

Reply:
@mickmister, using len causes fails on [check-mattermost-vet](https://app.circleci.com/pipelines/github/mattermost/mattermost-server/37355/workflows/c0c86552-960f-48f4-a991-5647ca24b675/jobs/324940)